components/compliance-service/reporting/relaxting/assets.go

```diff
@@ -0,0 +1,174 @@
+package relaxting
+
+import (
+	"context"
+	"github.com/chef/automate/components/compliance-service/ingest/ingestic/mappings"
+	"github.com/chef/automate/components/compliance-service/utils"
+	"github.com/olivere/elastic/v7"
+	"github.com/sirupsen/logrus"
+	"strings"
+	"time"
+)
+
+//getTotalAssets Get Total Number of documents from the comp-*-run-info
+func (backend ES2Backend) getAssets(ctx context.Context, boolQuery *elastic.BoolQuery) (int32, error) {
+	myIndex := mappings.ComplianceRunInfo.Index
+
+	client, err := backend.ES2Client()
+	if err != nil {
+		logrus.Errorf("Cannot connect to ElasticSearch: %v", err)
+		return 0, err
+	}
+
+	countService := elastic.NewCountService(client)
+
+	count, err := countService.Query(boolQuery).Index(myIndex).Do(ctx)
+	if err != nil {
+		logrus.Errorf("Cannot create client for count assets with error %v", err)
+		return 0, err
+	}
+
+	return int32(count), nil
+}
+
+//getFiltersQueryForAssetFilters Get Filteres query for the assets
+func (backend ES2Backend) getFiltersQueryForAssetFilters(filters map[string][]string) *elastic.BoolQuery {
+	utils.DeDupFilters(filters)
+	boolQuery := elastic.NewBoolQuery()
+
+	// These are filter types where we use ElasticSearch Term Queries
+	filterTypes := []string{"environment", "organization", "chef_server", "chef_tags",
```

Comment:
I see this list somewhere in the code, we should maintain all these things in one place only, so that 
 we can add/remove to the list easily, please define this list as constant and use it.